backend/users/views/user.py

```diff
@@ -124,6 +136,7 @@ def create(self, request, *args, **kwargs):
         role = request.data.get("role")
 
         _validate_channel(channel)
+        _validate_username(username)
```

Comment:
At the end having a DRF serializer would be nice here. But it could be done later.

Reply:
Agreed, but I didn't want to rewrite the whole view.